draft-ietf-quic-qpack.md

```diff
@@ -100,10 +100,13 @@ which might not yet have been processed by the recipient.  Such references
 are called "vulnerable," because the loss of a different packet can keep
 the reference from being usable.
 
-The encoder can choose on a per-header-block basis whether to favor higher
+The decoder can signal that it is willing to process vulnerable references by
```

Comment:
I still don't like the term "vulnerable references" here.  The point is that the decoder is willing to block on incomplete or in-progress header table updates or not.  "Vulnerable" as a concept isn't helpful, especially here.